libs/orchestrator-ui-components/src/lib/components/Tree/Tree.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { TreeBranch } from './TreeBranch';
+
+export const Tree = ({ data }) => {
```

Comment:
`Tree` component needs to be covered by both flavours of UI integration test(s) namely Jest snapshot tests and Storybook stories towards avoiding unintentional regressions both in the rendered HTML and the UI itself respectively.